tingbot/error.py

```diff
@@ -43,11 +43,7 @@ def sad_tingbot_image():
 
 
 def error_screen(exc_info):
-    surface = pygame.display.get_surface()
-    if not surface:
-        return
-
-    screen = graphics.Surface(surface)
+    screen = graphics.screen
 
```

Comment:
We could indeed use this (can't seem to find it yet - has it been pushed onto github?), but this leaves a larger question. Should the simulator be shown if the `screen` object is not used? I can imagine having a non-visible app that just responds to button presses ... in which case should we initialise the simulator image on startup (so the user can at least press buttons), rather than when `screen.surface` is accessed?